src/components/App.js

```diff
@@ -17,6 +21,10 @@ const App = () => (
   <Router>
     <div>
       <Route exact path="/" component={Home} />
+      <Route path="/history" component={History} />
+      <Route path="/examinations" component={Examinations} />
+      <Route path="/extras" component={Extras} />
+      <Route path="/stats" component={Statistics} />
```

Comment:
For each route, please add `strict equal`, e.g. `<Route strict equal path="/history" component={History} />`, otherwise we have multiple components rendering at certain endpoints.

Reply:
done